command/pr.go

```diff
@@ -26,7 +29,8 @@ func init() {
 	prCmd.AddCommand(prCloseCmd)
 	prCmd.AddCommand(prReopenCmd)
 	prCmd.AddCommand(prMergeCmd)
-	prMergeCmd.Flags().BoolP("merge", "m", true, "Merge the commits with the base branch")
+	prMergeCmd.Flags().BoolP("delete-branch", "d", true, "Delete the local branch after merge")
+	prMergeCmd.Flags().BoolP("merge", "m", false, "Merge the commits with the base branch")
```

Comment:
```suggestion
	prMergeCmd.Flags().BoolP("Merge", "m", false, "Merge the commits with the base branch")
```

Reply:
I don't think we want to upper case these; they are options like `--merge`.

Reply:
oh oops I probably found the wrong thing 🤦‍♀️